common/src/IO/MapFileSerializer.cpp

```diff
@@ -220,11 +220,14 @@ namespace TrenchBroom {
                 case Model::MapFormat::Standard:
                     return std::make_unique<QuakeFileSerializer>(stream);
                 case Model::MapFormat::Quake2:
-                    // TODO 2427: Implement Quake3 serializers and use them
+                    // TODO 2427: Implement Quake3 serializers... needs
```

Comment:
I don't think this adds much information -- we have all of this in the corresponding github issues (I hope). If anything is missing there, it should be added as a comment. Changes like these are adding noise to the commits.